### PR TITLE
feat:  Update Profile Avatar,  Static UI

### PR DESCRIPTION
**Pull request by Vishruta-Patil**
- Profile Avtar update and fetch
- UI : 
comments - comment card, create comment card
revamped UI of post card

### CODE REVIEW

File: src/components/Post/CreatePost.tsx [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import avatar from "assets/avatar.png";
+import { Avatar } from "components/common/avatar/Avatar";
 import { HeroBtn } from "components/common/button/HeroBtn";
 
 const CreatePost = () => {
   return (
     <div className="bg-slate-200 p-3 my-9 md:m-9 m-4 relative">
       <div className="flex space-x-3 items-center mb-9">
-        <img src={avatar} alt="avatar" className="h-16 w-16 rounded-full" />
+        {/* <img src={avatar} alt="avatar" className="h-16 w-16 rounded-full" /> */}
```
Comment (AmishMishra11) [inline src/components/Post/CreatePost.tsx:9]: remove comments if you are not using this

**Review by AmishMishra11 — APPROVED** [review]
When the user logs in for the first time there are 2 toasts saying the login is successful so try to remove that bug
Great work on UI

**Review by prathmeshjagtap — APPROVED** [review]
Code is Clean and well structured you have followed all the best practices

**Comment by Vishruta-Patil** [comment]
> When the user logs in for the first time there are 2 toasts saying the login is successful so try to remove that bug Great work on UI

Thanks for the review, I checked but it is working well for me. It is showing only one toast after logging in. If possible then please check one more time.

**Pull request merged**